main.go

```diff
@@ -42,8 +42,8 @@ var quitter = make(chan struct{})
 
 var logLevel = "INFO"
 var debugLogs = flag.Bool("debug", false, "Enable debug logs?")
-var dbDialect = flag.String("db-dialect", "sqlite3", "Database dialect (sqlite3 or postgres)")
-var dbAddress = flag.String("db-address", "file:whatsmeow.db?_foreign_keys=on", "Database address")
+var dbDialect = flag.String("db-dialect", "sqlite", "Database dialect (sqlite or postgres)")
+var dbAddress = flag.String("db-address", "file:whatsmeow.db?_foreign_keys=on&_pragma=foreign_keys(1)", "Database address")
```

Comment:
`_foreign_keys=on` is specific to the mattn implementation of sqlite. We can omit it.